src/utils/getReactNativeVersion.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function getReactNativeVersion(cwd: string) {
+  try {
+    const pak = JSON.parse(
+      fs.readFileSync(path.join(cwd, 'package.json')).toString()
+    );
+
+    const deps = {
+      ...(pak.dependencies || {}),
+      ...(pak.devDependencies || {}),
```

Comment:
I would reorder this to
```js
...(pak.devDependencies || {})
...(pak.dependencies || {})
...(pak.peerDependencies || {})
```
`devDependencies` should have lesser priority than `dependencies`

Reply:
I just copied existing https://github.com/callstack/haul/blob/master/src/utils/isReactNativeProject.js

Reply:
In that case you can leave it as it is.